### PR TITLE
Properties widget updates

### PR DESCRIPTION
**Pull request by kisantia**
Updates for properties widget to look more like mockup:
- Remove title "SERVER DASHBOARD" because it isn't in mockups 
- Remove ellipses because refresh action has been moved to the toolba
- Move collapse chevron icon to bottom center of widget instead of current top right location 

expanded:
image

collapsed:
image

### CODE REVIEW

**Comment by alanrenmsft** [comment]
when it is collapsed, I think we should show the title and the expand button. Please take a look at the Azure portal.

**Comment by kisantia** [comment]
updated screenshots to show collapsed properties widget with the title

File: src/sql/workbench/contrib/dashboard/browser/containers/dashboardHomeContainer.component.ts [hunk 1]
```diff
@@ -18,16 +18,22 @@ import { TabChild } from 'sql/base/browser/ui/panel/tab.component';
...
 	providers: [{ provide: TabChild, useExisting: forwardRef(() => DashboardHomeContainer) }],
 	template: `
 		<div class="fullsize" style="display: flex; flex-direction: column">
 			<div scrollable [horizontalScroll]="${ScrollbarVisibility.Hidden}" [verticalScroll]="${ScrollbarVisibility.Auto}">
-				<dashboard-widget-wrapper #propertiesClass *ngIf="properties" [collapsable]="true" [_config]="properties"
-					style="padding-left: 10px; padding-right: 10px; display: block; flex: 0" [style.height.px]="_propertiesClass?.collapsed ? '30' : '90'">
-				</dashboard-widget-wrapper>
+				<div #propertiesContainer>
+					<dashboard-widget-wrapper #propertiesClass *ngIf="properties" [collapsable]="true" [bottomCollapse]="true" [toggleMore]="false" [_config]="properties"
+						style="padding-left: 10px; padding-right: 10px; display: block; flex: 0;" [style.height.px]="_propertiesClass?.collapsed ? '30' : '90'">
```
Comment (alanrenmsft) [inline src/sql/workbench/contrib/dashboard/browser/containers/dashboardHomeContainer.component.ts:34]: css class?

File: src/sql/workbench/contrib/dashboard/browser/contents/dashboardWidgetWrapper.component.ts [hunk 2]
```diff
@@ -72,6 +75,14 @@ export class DashboardWidgetWrapper extends AngularDisposable implements OnInit
 		}
 	}
 
+	public get showTitle(): boolean {
```
Comment (alanrenmsft) [inline src/sql/workbench/contrib/dashboard/browser/contents/dashboardWidgetWrapper.component.ts:78]: do you really need this property? isn't it related to the collapsed state?

File: src/sql/workbench/contrib/dashboard/browser/contents/dashboardWidgetWrapper.component.html [hunk 3]
```diff
@@ -18,4 +18,5 @@
 		<ng-template component-host>
 		</ng-template>
 	</ng-template>
+	<span #bottomActionbar style="flex: 0 0 auto; align-self: center; margin-top: -20px"></span>
```
Comment (alanrenmsft) [inline src/sql/workbench/contrib/dashboard/browser/contents/dashboardWidgetWrapper.component.html:21]: while we are on this file, do you mind use css classes for the styles?

**Comment by alanrenmsft** [comment]
the bottom padding of the button is not consistent between the two modes, the button should be horizontally center aligned when expanded and also vertically center aligned when collapsed.

File: src/sql/workbench/contrib/dashboard/browser/contents/dashboardWidgetWrapper.component.ts (hunk 2 above)
Comment (kisantia) [reply src/sql/workbench/contrib/dashboard/browser/contents/dashboardWidgetWrapper.component.ts:78]: yes this is needed if a widget wants to always show the title(when collapsed and expanded), compared to how properties widget only wants to show the title when collapsed.

**Comment by kisantia** [comment]
vertically center aligned collapse: 
image

image

**Pull request merged**